src/main/java/sirius/db/jdbc/WrappedPreparedStatement.java

```diff
@@ -42,24 +42,32 @@
  * Wrapper for {@link PreparedStatement} to add microtiming.
  */
 class WrappedPreparedStatement implements PreparedStatement {
+
+    private static final Duration LONG_RUNNING_QUERY_OPERATION = Duration.ofMinutes(15);
```

Comment:
Shouldn't this be 5 minutes if trying to match 300s? 🧐

Reply:
No the actual socket timeout is specified in the config. However, as this is a hardcoded constant (for now), we permit an even longer interval...